cmd/atlas/docv2.go

```diff
@@ -0,0 +1,50 @@
+package main
```

Comment:
script to generate and maintain importable markdown blocks containing the CLIs built-in help text.